core/src/sql/value/value.rs

```diff
@@ -70,7 +71,7 @@ impl From<&Tables> for Values {
 	}
 }
 
-#[revisioned(revision = 1)]
+#[revisioned(revision = 2)]
```

Comment:
I don't think we need to use revision. We just need to ensure that the variant is placed at the end (before line 124).